docs/components/movement-sensor/merged.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Configure an merged movement sensor"
+linkTitle: "merged"
+weight: 40
+type: "docs"
+description: "Configure an merged movement sensor."
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand
+---
+
+The `merged` movement sensor model supports a movement sensor that allows you to get multiple types of [`Readings`](/components/movement-sensor/#getreadings), including angular and linear velocity, from *one* movement sensor {{< glossary_tooltip term_id="resource" text="resource" >}} on your robot.
+
+This allows you to aggregate the API methods supported by multiple sensors into a singular sensor client, effectively merging the models of the individual resources.
+
+In other words, the models of the `merged` sensors are combined, making the movement sensor API methods each aggregated model supports available on the `merged` model.
+
+This is especially useful if you want to get readings of position and orientation *or* linear and angular velocity at the same time, which are normally separately supported and returned by [`GPS`](/components/movement-sensor/#configuration) or [`IMU`](/components/movement-sensor/#configuration) models, respectively.
+
+{{% alert title="Usage Tip: Navigation Service" color="tip" %}}
```

Comment:
I think rand wants it a tip (see comment thread below) so not addressing

Reply:
It is weird here but this is quite possibly necessary for usage rn from what I understand?

Reply:
I see. What if this lives in the navigation service docs, not here? Would that work?

If this stays here, I'm still not sure about creating a specialized title for this tip instead of using "Tip" as the title for consistency across the docs.